satpy/etc/composites/ahi.yaml

```diff
@@ -2,24 +2,11 @@ sensor_name: visir/ahi
 
 modifiers:
   rayleigh_corrected:
-    modifier: !!python/name:satpy.modifiers.PSPRayleighReflectance
-    atmosphere: us-standard
-    aerosol_type: marine_clean_aerosol
-    prerequisites:
-    - wavelength: 0.64
-      modifiers: [sunz_corrected]
-    optional_prerequisites:
-    - satellite_azimuth_angle
-    - satellite_zenith_angle
-    - solar_azimuth_angle
-    - solar_zenith_angle
-
-  no_aerosol_rayleigh_corrected:
     modifier: !!python/name:satpy.modifiers.PSPRayleighReflectance
     atmosphere: us-standard
     aerosol_type: rayleigh_only
     prerequisites:
-    - name: B03
+    - wavelength: B03
```

Comment:
How can this work?

Reply:
Maybe I generated the image and then made this change right before committing. You know, like a bad TDD developer.